easybuild/easyconfigs/g/GDCM/GDCM-2.8.7-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'GDCM'
+version = '2.8.7'
+
+homepage = 'http://gdcm.sourceforge.net/'
+description = """Grassroots DICOM (GDCM) is an implementation of the DICOM standard designed to be open source so
+that researchers may access clinical data directly. GDCM includes a file format definition and a network
+communications protocol, both of which should be extended to provide a full set of tools for a researcher
+or small medical imaging vendor to interface with an existing medical database."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [('http://sourceforge.net/projects/gdcm/files/%(namelower)s%202.x/%(name)s%20%(version)s', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7a08baa93e90bce17d9999d59b95876808801a287812348e27a23decb1ebc58c']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```

Comment:
@crubb No need to use `dummy` binutils here, please use `('binutils', '2.30')` instead